flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/schema/TimeIndicatorRelDataType.scala

```diff
@@ -34,6 +34,8 @@ class TimeIndicatorRelDataType(
     originalType.getSqlTypeName,
     originalType.getPrecision) {
 
+  override def computeDigest(): Unit = this.digest = toString
```

Comment:
Remove `this.digest = `

Reply:
oops missed this one. will make the final change along with the CALCITE JIRA id